search/geocoder.cpp

```diff
@@ -667,16 +669,31 @@ void Geocoder::FillLocalitiesTable(BaseContext const & ctx)
       {
         ++numCities;
 
-        auto const center = feature::GetCenter(ft);
-        auto const population = ftypes::GetPopulation(ft);
-        auto const radius = ftypes::GetRadiusByPopulation(population);
-
         City city(l, Model::TYPE_CITY);
-        city.m_rect = MercatorBounds::RectByCenterXYAndSizeInMeters(center, radius);
+
+        CitiesBoundariesTable::Boundaries boundaries;
+        bool haveBoundary = false;
+        if (m_citiesBoundaries.Has(ft.GetID()) && m_citiesBoundaries.Get(ft.GetID(), boundaries))
```

Comment:
It seems that you may safely remove first call to Has, because Get() implies Has().